modules/cli/src/main/scala/scala/cli/commands/new/New.scala

```diff
@@ -0,0 +1,57 @@
+package scala.cli.commands.`new`
+
+import caseapp.core.RemainingArgs
+import dependency.*
+
+import scala.build.internal.{Constants, OsLibc, Runner}
+import scala.build.options.{BuildOptions, JavaOptions}
+import scala.build.{Artifacts, Logger, Positioned}
+import scala.cli.commands.ScalaCommand
+import scala.cli.commands.shared.{CoursierOptions, HelpCommandGroup}
+
+object New extends ScalaCommand[NewOptions] {
+  override def group: String = HelpCommandGroup.Main.toString
+
+  override def scalaSpecificationLevel = SpecificationLevel.IMPLEMENTATION
+
+  val giter8Dependency =
+    Seq(dep"${Constants.giter8Organization}::${Constants.giter8Name}:${Constants.giter8Version}")
+
+  override def runCommand(options: NewOptions, remainingArgs: RemainingArgs, logger: Logger): Unit =
+    val fetchedGiter8 = Artifacts.fetch(
+      Positioned.none(giter8Dependency),
```

Comment:
@lwronski hi! Thank you for the help :D. I kinda just worked with the example you gave, because `Artifacts.fetch` is all function signature.

With my current code I'm running into
```
Got Scala dependency org.foundweekends.giter8::giter8_2.13:0.16.2, but no Scala version is provided
```

I'm not sure what the best way is to "supply" the scala version

Reply:
I will suggest solution in the comments that should resolve your problems.